addon/components/connect.js

```diff
@@ -92,7 +98,11 @@ export default (stateToComputed, dispatchToActions=() => ({})) => {
 
       didUpdateAttrs() {
         this._super(...arguments);
-        this.handleChange();
+        // Components only have a handleChange method if it is subscribed
+        // to redux changes.
+        if (this._handleChange) {
```

Comment:
If you don't mind I'll remove these 2 comments as they are self evident above (reading the source for 10 min will expose why you added this conditional)